src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void serviceInitialization() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void userStorageInitialization() {
+        user = new User(1L,"valid_login", "123456", 40);
+        Storage.people.addAll(List.of(new User(2L,"LarsUrlich","myolddrums",60),
+                new User(2L, "JamesHetfield","mastermaster",61),
+                new User(3L,"KirkHammet","Imtoofastforyou",60),
+                new User(4L,"JasonNewsted","imnothere777",99)));

```

Comment:
If you want to do it this way, you should clear the Storage before each (do you really need this many users? Isn't one enough?): 
```suggestion
        Storage.people.clear();
        user = new User(1L,"valid_login", "123456", 40);
        Storage.people.addAll(List.of(new User(2L,"LarsUrlich","myolddrums",60),
                new User(2L, "JamesHetfield","mastermaster",61),
                new User(3L,"KirkHammet","Imtoofastforyou",60),
                new User(4L,"JasonNewsted","imnothere777",99)));

```

Reply:
ok, done, I moved the test @AfterEach just below @BeforeEach, to make code more clearer.